scan_node:
  ros__parameters:
    use_sim_time: true
    referece_frame: "geometric_unicycle"
    scan_topic: "/scan_local"
    pub_topic: "/compensated_pc2"
    imu_topic: "/head_imu"
    filter_z_low: 0.2
    filter_z_high: 2.5
    close_threshold: 0.5
    imuVel_x_threshold: 0.4
    imuVel_y_threshold: 0.4
    ms_wait: 400.0
    robot_on_crane: true